bin/mixserv_daemon.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/sh
+
+# Hivemall: Hive scalable Machine Learning Library
+#
+# Copyright (C) 2015 Makoto YUI
+# Copyright (C) 2013-2015 National Institute of Advanced Industrial Science and Technology (AIST)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+usage="Usage: mixserv_daemon.sh (start|stop|status)"
+
+# If no args specified, show usage
+if [ $# -ne 1 ]; then
+  echo $usage
+  exit 1
+fi
+
+if [ "$HIVEMALL_HOME" == "" ]; then
+  echo env HIVEMALL_HOME not defined
+  exit 1
+fi
+
+# Load global variables
+. "$HIVEMALL_HOME/conf/mixserv_env.sh"
+
+# Load a version number from a VERSION file
+VERSION=`cat $HIVEMALL_HOME/VERSION`
+
+MIXSERV_PID_DIR="/tmp"
+MIXSERV_PID_FILE="$MIXSERV_PID_DIR/hivemall-mixserv-$VERSION-$USER.pid"
+MIXSERV_LOG_DIR="$HIVEMALL_HOME/logs"
+MIXSERV_LOG_FILE="$MIXSERV_LOG_DIR/hivemall-mixserv-$VERSION-$USER-$HOSTNAME.out"
+
+rotate_mixserv_log() {
```

Comment:
Why not just use RollingFileAppender?
http://2min2code.com/articles/log4j_intro/rolling_file_on_size_prop